site/cypress/e2e/polls.spec.js

```diff
@@ -279,6 +283,62 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('.radio').eq(3).contains('Answer 2');
         cy.get('.radio > input').eq(3).should('not.be.checked');
 
+        // log into instructor and edit histogram, answer release and close the poll
+        cy.logout();
+        cy.login();
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().click();
+        cy.contains('Poll Cypress Test').siblings(':nth-child(1)').children().click();
+        cy.url().should('include', 'sample/polls/editPoll');
+        cy.get('#breadcrumbs > :nth-child(7) > span').should('have.text', 'Edit Poll');
+        cy.get('#student-histogram-release-setting').select('when_ended');
+        cy.get('#student-answer-release-setting').select('when_ended');
+        cy.get('button[type=submit]').click();
+        cy.wait(1000);
```

Comment:
This wait is probably unnecessary, and slows down the test suite quite a bit.  It also has the potential to result in a flaky test if the test runner is running slowly sometime.  The other waits should be eliminated as well.

Reply:
The final way is reserved there but reduced to 500 to ensure the test runs smoothly.